docs/detections/api/exceptions/api-find-exception-containers.asciidoc

```diff
@@ -34,15 +34,22 @@ See {ref}/search-request-body.html#request-body-search-search-after[Search After
 associated with a {kib} space or available in all spaces (`agnostic` or
 `single`).
 
-|`filter` |String a|Filters the returned results according to the value of the
-specified field, using the `<field name>:<field value>` syntax, where
-`<field name>` can be:
+|`filter` |String a|Filters the returned results according to the value of the specified field. 
+
+Uses the `<so type.field name:field value>` syntax, where `<so type>` can be:
+
+* `exception-list`: Specify a space-aware exception list.
+* `exception-list-agnostic`: Specify an exception list that is shared across spaces.
+
+And `<field name>` can be:
```

Comment:
Maybe omit the angle brackets -- it's a little unclear if they're literal syntax or functioning more like quotation marks, and sometimes they're used for the whole string and sometimes just for a part of it. The monospace format already makes the values stand out from normal text.

```suggestion
Uses the `so type.field name:field value` syntax, where `so type` can be:

* `exception-list`: Specify a space-aware exception list.
* `exception-list-agnostic`: Specify an exception list that is shared across spaces.

And `field name` can be:
```